zer0-os/zOS
# feat: position feed and chat containers and add central divider when current channel is social channel

### What does this do?
- position the feed and chat containers and add central divider when current channel is social channel

### Why are we making this change?
- as per design requirement

### How do I test this?
- run ui > enableChannels feature flag > create social channel group > check layout > even container widths

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

<img width="1182" alt="Screenshot 2024-08-16 at 01 33 02" src="https://github.com/user-attachments/assets/336efea3-d251-4b0c-98b3-dfe4edea5726">
